tests/testthat/test_strip_html.r

```diff
@@ -0,0 +1,41 @@
+library(dplyr)
+library(testthat)
+
+test_that("all <> and values between are removed", {
+
+
+  cols <- c("What is your favorite color?", "Keep  value")
+
+  expect_equal(mtcars %>%
+                 select("What is your <strong>favorite</strong> color?" = mpg,
+                        "Keep <this> value" = cyl) %>%
+                 strip_html() %>%
+                 colnames(.),
+               cols)
+  })
+
+
+test_that("ignore values are kept", {
+
```

Comment:
re: my comment above, where I questioned whether the `check_`ignore` values would not match and thus print a warning even though they're present -- if this test doesn't throw a warning, then maybe my comment above is wrong.

Reply:
Yep ignore this, I was missing the `|` piece as or